### ISSUE TITLE
soft teacher loss different in paper and code.

### ISSUE THREAD

**Issue by yangkai798** [microsoft/SoftTeacher#19]
bg_score = torch.cat([_score[:, -1] for _score in _scores])
            assigned_label, _, _, _ = bbox_targets
            neg_inds = assigned_label == self.student.roi_head.bbox_head.num_classes
            bbox_targets[1][neg_inds] = bg_score[neg_inds].detach()
        loss = self.student.roi_head.bbox_head.loss(
            bbox_results["cls_score"],
            bbox_results["bbox_pred"],
            rois,
            *bbox_targets,
            reduction_override="none",
        )
        loss["loss_cls"] = loss["loss_cls"].sum() / max(bbox_targets[1].sum(), 1.0)

the soft teacher loss different in paper and code.

![image](https://user-images.githubusercontent.com/33416945/133562637-63040f2d-c2b1-4ddd-89dd-63220695670f.png)

**Comment by MendelXu** [comment]
Yes, you are right. I think the equation in the paper is wrong. We will update it in our next version paper.
